### ISSUE TITLE
Removal of unmaintained mappings

### ISSUE THREAD

**Issue by Holzhaus** [mixxxdj/mixxx#12045]
I suggest the removal clearly broken mappings. These seem to be unmaintained and won't work properly which results in a bad impression for new users. We could announce it on the forums and/or our website and ask for help, so that users that actually do use them may step up and help fixing them.

Among those mappings are those who still use the `lfoDelay` control, which was removed (not just deprecated) in Mixxx 2.0.0:

- American-Audio-RADIUS-2000
- Behringer BCD2000
- Behringer BCD3000
- Behringer BCD3000 Advanced
- DJ-Tech i-Mix Reload
- Denon MC3000
- Denon-DN-SC2000
- Denon-MC3000
- Hercules-DJ-Console-Mk1-hid
- Hercules-DJ-Console-Mk2-hid
- Hercules-DJ-Console-Mk2
- Hercules-DJ-Console-RMX
- Hercules-DJ-Control-MP3-hid
- Hercules-DJ-Control-Steel
- Korg-nanoKONTROL-2
- Mixman DM2 (Linux)
- Numark MIXTRACK
- Numark Mixtrack Pro
- Numark Omni Control
- Numark Total Control
- Numark-Mixtrack-Pro
- Numark-V7
- Stanton SCS
- Stanton-SCS1d
- TrakProDJ iPad
- Vestax VCI-100
- Vestax VCI-100-3DEX
- Wireless DJ App
- Wireless-DJ
- us428

**Comment by daschuer** [comment]
An alternative could be to pop up a disclaimer "Outated mapping, Help wanted"
Maye there are still useful parts but that way no one will blame use.   

Just an idea, no demand.

**Comment by ronso0** [comment]
Good idea IMO. If it's just the Fx components that are broken a warning for 95% mapping is still better than no mapping.
We could add that warning to the mapping description.

I expect that especially users with old hardware will be grateful if Mixxx supports them with a somewhat working mapping

**Comment by fwcd** [comment]
I'd probably just delete the functions that depend on removed APIs and leave the rest up to the user. Fixing a broken mapping is in most cases probably easier than remapping from scratch.

Adding a short note/warning that the mapping is unmaintained sounds good.

**Comment by joshpatten** [comment]
I just submitted a PR with an updated mapping for the Numark Mixtrack Pro, could you please not kill that one?

**Comment by ronso0** [comment]
Don't worry, I think no one had this PR on the radar let alone started working on it ; )

Also, I think we agreed to _not_remove mappings but add a warning to the mapping description.
